<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEn2CnKeyForm</class>
 <widget class="QWidget" name="CEn2CnKeyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widgetTitle" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>译文：英文--&gt;中文</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetEn" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelEnTitle">
             <property name="text">
              <string>英文：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEditEn">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetCn" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelCnTitle">
             <property name="text">
              <string>中文：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEditCn">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>192</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widgetKey" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pbtnLoadKey">
           <property name="text">
            <string>加载译文</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnAddKey">
           <property name="text">
            <string>增加译文</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnEditKey">
           <property name="text">
            <string>修改译文</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnDelKey">
           <property name="text">
            <string>删除译文</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnExportKey">
           <property name="text">
            <string>导出译文</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnImportKey">
           <property name="text">
            <string>导入译文</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>116</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnTranslateFile">
           <property name="text">
            <string>翻译单个文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnTranslatePath">
           <property name="text">
            <string>翻译单个目录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnTranslateAllPath">
           <property name="text">
            <string>翻译所有目录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableViewKey"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
